node_modules/.bin/acorn

```diff
@@ -0,0 +1 @@
+../acorn/bin/acorn
```

Comment:
You should never add node_modules to the git repository.

`node_modules` are your dependencies, and dependencies are not the code, that you write. Dependencies are easily recreated by `npm install` command, so there is no need to track them in the source code repository.

There is a special `.gitignore` file, which is used to specify the files and folders, ignored by git. Here is an [article about .gitignore](https://chaseonsoftware.com/2015/07/my-gitignore-conventions/)

But you are in a problem right now, because node_modules are already inside the git repository, and they will not be deleted automatically. Take a look at [this StackOveflow answer](https://stackoverflow.com/questions/50675829/remove-node-modules-from-git-in-vscode) to find out how to delete an added folder to git.